client/dashboard/task-list/tasks/payments/payfast.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, Fragment } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Link } from '@woocommerce/components';
+import { ADMIN_URL } from '@woocommerce/wc-admin-settings';
+
+/**
+ * Internal dependencies
+ */
+import { recordEvent } from 'lib/tracks';
+
+export default class PayFast extends Component {
+  constructor( props ) {
+    super( props );
+  }
+
+  continue = () => {
+    recordEvent( 'tasklist_payment_connect_method', {
+      payment_method: 'payfast'
+    } )
+  }
+
+  render() {
+    const link = <Link
+      href={ `${ ADMIN_URL }/admin.php?page=wc-settings&tab=checkout&section=wc_gateway_payfast` }
```

Comment:
We can use `getAdminLink( 'admin.php?page=wc-settings&tab=checkout&section=wc_gateway_payfast' )` here.